index.md

```diff
@@ -17,9 +17,9 @@ header:
 excerpt: Microbiome Data Science
 ---
 
-## We study microbiome
+## We study microbiomes
 
-We are the Qiyun Lab at the Biodesign Center for Fundamental and Applied Microbiomics ([CFAM](https://biodesign.asu.edu/cfam)), Arizona State University ([ASU](https://www.asu.edu/)). We study the data science for microbiomics and multiomics. We are interested in the development of computational tools and resources to improve microbiome data analytics, the study of various host and environmental microbiomes and their influence on human health, and the investigation of the evolution of microbes and their hosts, with focus on vertical and horizontal genetic connections between organisms. We strive to expand understanding of the microbial world through its multiple levels: genes, genomes, lineages, communities, hosts, and environments.
+We are the Qiyun Lab at the Biodesign Center for Fundamental and Applied Microbiomics ([CFAM](https://biodesign.asu.edu/cfam)) at Arizona State University ([ASU](https://www.asu.edu/)). We study the applications of data science for microbiomics and multiomics analyses. We are interested in the development of computational tools and resources to improve microbiome data analytics, the study of various host and environmental microbiomes and their influence on human health, and the investigation of the evolution of microbes and their hosts, with a focus on vertical and horizontal genetic connections between organisms. We strive to expand understanding of the microbial world through its multiple levels: genes, genomes, lineages, communities, hosts, and environments.
```

Comment:
How it reads to be right now is saying that we study data science derived from microbiomics and multiomics. I'm not sure if that was the original intent of the sentence.

My change flips the meaning. In this instance, "applications of" refers using any form of data science to solve problems. The meaning is that we take the principles of data science and use them to solve microbiomics and multiomics problems. So applications in this context is purely describing tangible use cases of data science principles. You could also rephrase it to something like "We develop and study applications of...". 

As for CFAM, you are correct from the meaning of the title. Fundamental = basic science, Applied can mean using existing tools to solve problems. But applied also has a broader meaning where you are taking that basic science and applying it to solve real world problems. For example, we make a finding about the gut microbiome after an omics analysis and you "apply" those findings to a medical treatment. In this instance apply/application means we are taking something we learned and using that to solve a problem.

Reply:
Thanks for detailing your thoughts on the wording. I see your point about "application". I am still not fully sold though. Because we _not only_ use data science principles to solve problems, but we also _create_ data science principles themselves, the latter of which (I hope!) differentiate us from people who use existing tools. More specifically, we develop equations, algorithms, programs and protocols which other people can take to make findings from their microbiome and meta-omics datasets. Therefore I just used a single "data science" phrase.

(Maybe you made edits but I couldn't see from the pull request. Sorry... See my comment in the end.)